pytest.ini

```diff
@@ -1,5 +1,5 @@
 [pytest]
-addopts = --ds=config.settings.test --reuse-db --disable-socket --ignore=e2e_tests/
+addopts = --ds=config.settings.test --reuse-db --disable-socket --ignore=e2e_tests/ --ignore=marklogic/
```

Comment:
why do we want pytest to ignore the marklogic folder?

Reply:
aha the marklogic folder only exists in CI when we checkout the marklogic repo: there's a file called `test_` something in there which isnt actually a pytest test, and which causes pytest to fail.

The marklogic tests are still run when we merge marklogic to main, so i don't think it's necessary to run them here